src/k8s_client/k8s_client.go

```diff
@@ -485,7 +485,7 @@ func (c *k8sClient) GetPodLogsAsBuffer(namespace string, podName string, sinceSe
 }
 
 func (c *k8sClient) IsMetalProvisioningExists() (bool, error) {
-	u := &unstructured.UnstructuredList{}
+	u := &unstructured.Unstructured{}
```

Comment:
@yevgeny-shnaidman i'm not sure about this part, do you know if it's tested somewhere?

Reply:
it was added for 4.7, and tested with 4.7 on real baremetal cluster (3 lab machines). Since then i don't think that we touched it.